src/wallet/rpcwallet.cpp

```diff
@@ -5014,17 +5014,20 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee ) ( privacyPolicy )\n"
             "\nSend multiple times. Amounts are decimal numbers with at most 8 digits of precision."
             "\nChange generated from one or more transparent addresses flows to a new transparent"
-            "\naddress, while change generated from a shielded address returns to itself."
-            "\nWhen sending coinbase UTXOs to a shielded address, change is not allowed."
-            "\nThe entire value of the UTXO(s) must be consumed."
-            + strprintf("\nBefore Sapling activates, the maximum number of zaddr outputs is %d due to transaction size limits.\n", Z_SENDMANY_MAX_ZADDR_OUTPUTS_BEFORE_SAPLING)
```

Comment:
This was the only remaining use of `Z_SENDMANY_MAX_ZADDR_OUTPUTS_BEFORE_SAPLING`, so that definition should be deleted.

Reply:
[force-pushed](https://github.com/zcash/zcash/compare/27594259147f9c8eafc66f1085fc81adb5d8ab10..6d8d82f18b0acd3efcc2761c7200d9949dff43e2) to fix this.